packages/serp-report/src/background/serp-report.js

```diff
@@ -8,7 +8,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
-
+import * as tldts from '../../../../tldts/dist/index.esm.min.js';
```

Comment:
this I'm not sure it should be here. 
shouldn't we rely on the `tldts` to be in scope already?

Reply:
this fill force `ghostery-extension` to include vendor version of tldts to main bundle

Reply:
As we use ES modules, the tldts can be imported as a module. And it is in the `worker/index.js` in DNR extension. As we use there ES module for tldts, we must do it here as well.

Reply:
`ghostery-extension` uses `tldts-experimental` which is imported directly from node_modules via webpack.

perhaps `ghostery-dnr-extension` should also use `tldts-experimental` but that does not resolve the problem as this code will still import same code from another place. 
I don't see a way other than assuming it is available in the context.

Reply:
Finally, I moved to `globalThis.tldts` for the compatibility for now. After moving to bare imports (hopefully), it might be changed.